test/it/helpers/browser.js

```diff
@@ -122,3 +211,41 @@ export function isFirefoxAfter86 () {
   return browser.capabilities.browserName === 'firefox' &&
     parseInt(browser.capabilities.browserVersion.substring(0, 2)) > 86
 }
+
+export function redefineSetTimeout () {
```

Comment:
Read the linked issue for details.

tldr: ios on browserstack signals that it's w3c spec compliant, but the wdio team doesn't give a shit. sooooo ... 👇